docs/manage/partitions/flex/flex-pricing-faq.md

```diff
@@ -75,4 +75,15 @@ The table below should give you a sense of how the number of partitions you use
 We recommend you configure partitions to have less than 5 TB per day flowing into them. This practice:
 
 * Reduces the cost of each query.
-* Results in faster queries, since less data has to be scanned.  
\ No newline at end of file
+* Results in faster queries, since less data has to be scanned.  
+
+## How to optimize the query using default scope?
+
+Default scope allows you to include or exclude the partitions in the search query, helping you to optimize the search cost. For example, consider if a query needs to run through 10 partitions which consumes about 10 GB of search data, narrowing the search query using the default scope can reduce the time and cost. You can achieve this by selecting or deselecting the **Include this partition in default scope** checkbox when creating the partition. Let's say, out of 10 partitions you excluded two partitions, now, when you run a query which does not have `_index`, the search will only consider the included partitions, reducing the amount of data scanned and lowering the cost.
+
+When partitions are marked as included and `_index` is not referred to in the query, all the included partitions will be considered for the search query by default. Default scope can also come to use when `AND` or `OR` conditions are referred to in the query with `_index`. For example, consider you have three partitions namely, Partition A (Excluded), Partition B (Included), and Partition C (Included). Below are some scenarios:
+
+- When you run the query without referring to `_index`, only Partition B and Partition C will be considered for the query. 
+- When you run a query referring to `_index=Partition A`, only Partition A will be considered for the query. 
+- When you run the query, `_index=Partition A OR _index=Partition B`, only Partition A and Partition B will be considered for the query. 
+- When you run the query, `_index=Partition A OR error`, Partition A, Partition B, and Partition C will be considered for the query. 
```

Comment:
```suggestion
- However, When you run the query, `_index="Partition A" OR error`, Partition A, Partition B, and Partition C will be considered for the query, because the error keyword might be present in Partition B or C as well. So in cases, like this, we need to scan all three of them. 
```